db/db_impl/db_impl_write.cc

```diff
@@ -1140,13 +1145,13 @@ Status DBImpl::PreprocessWrite(const WriteOptions& write_options,
   return status;
 }
 
-WriteBatch* DBImpl::MergeBatch(const WriteThread::WriteGroup& write_group,
-                               WriteBatch* tmp_batch, size_t* write_with_wal,
-                               WriteBatch** to_be_cached_state) {
+Status DBImpl::MergeBatch(WriteBatch** merged_batch,
```

Comment:
Is `merged_batch` output argument? If so, it should be placed after all input arguments.